### PR TITLE
Add cat class file

### PR DESCRIPTION
**Pull request by gitkseniya**
I added one markdown file with an image of my baby (of course).
Please review the class instances and methods I came up with and let me know how I did.
Thank you Aaron!

2ocwvi

### CODE REVIEW

File: cat_class.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* teach_fetching (adds value to the array responsibilities): "love", "meow", "sleep", "eat", "throw things off counter", "fetch"]
+
+![Meet Fallon](assets/fallon.JPG)
+
+
+Class Instance: Moose
+* name: "Moose"
+* age: 4
+* long_nails: no
+* responsibilities: ["meow", "sleep", "eat", "be a yard bouncer"]
```
Comment (Abdeboskey) [inline cat_class.md:30]: Hahaha be a yard bouncer 😹

File: cat_class.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Methods:
+* clip_nails
+* teach_fetching
+
+
+Class Instance: Fallon
+* name: "Fallon"
+* age: 8
+* long_nails: yes
```
Comment (Abdeboskey) [inline cat_class.md:17]: In Ruby, do you use yes/no for Boolean or true/false?

**Review by Abdeboskey — APPROVED** [review]
These are perfect cat instances. Do you know enough Ruby at this point to make a Ruby file and actually write these classes/instances? 🤓 you're doing great KK 🖤

**Comment by gitkseniya** [comment]
I don't know how to do it yet, but why don't we try? I'll use a tutorial :)

*Kseniya Kudzelich*

**Pull request merged**